airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/destination_async/state/AsyncStateManagerTest.java

```diff
@@ -25,10 +27,12 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import org.apache.commons.lang.RandomStringUtils;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class AsyncStateManagerTest {
```

Comment:
made it so this class compiles, but didn't actually try to get the tests to run since @davinchia is working on them currently.

Reply:
we will want to add some tests specific to the memory mgmt.